global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "8.0.100-preview.7.23376.3"
+    "version": "8.0.100-rc.1.23455.8"
```

Comment:
@Tratcher rc2 is about to be released and we're just getting rc1 here. How do we get rc2 here quicker to get a build with the IIISEnvironmentFeature I added?